sale_pricelist_technical/i18n/fr.po

```diff
@@ -0,0 +1,61 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#   * sale_pricelist_technical
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-05-12 14:48+0000\n"
+"PO-Revision-Date: 2022-05-12 14:48+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: sale_pricelist_technical
+#: model:ir.model.fields,help:sale_pricelist_technical.field_product_pricelist__is_technical
+msgid "Check this box if you want to prevent user to use this pricelist on the orders, or in the customer form. This is usefull for pricelist created only to compute other pricelists."
+msgstr "Cocher cette case si vous souhaitez empêcher les utilisateurs d'utiliser cette liste de prix dans les devis, ou sur le formulaire client. Ceci est utile pour les liste de prix créée uniquement pour le calcul d'autres liste de prix."
+
+#. module: sale_pricelist_technical
+#: model:ir.model,name:sale_pricelist_technical.model_res_partner
+msgid "Contact"
+msgstr ""
+
+#. module: sale_pricelist_technical
+#: model:ir.model.fields,field_description:sale_pricelist_technical.field_product_pricelist__is_technical
+msgid "Is Technical"
+msgstr "Est technique"
+
+#. module: sale_pricelist_technical
+#: model:ir.model,name:sale_pricelist_technical.model_product_pricelist
+#: model:ir.model.fields,field_description:sale_pricelist_technical.field_res_partner__property_product_pricelist
+#: model:ir.model.fields,field_description:sale_pricelist_technical.field_res_users__property_product_pricelist
+#: model:ir.model.fields,field_description:sale_pricelist_technical.field_sale_order__pricelist_id
+msgid "Pricelist"
+msgstr "Liste de prix"
+
+#. module: sale_pricelist_technical
+#: model:ir.model.fields,help:sale_pricelist_technical.field_sale_order__pricelist_id
+msgid "Pricelist for current sales order."
+msgstr "Liste de prix du bon de commande actuel"
+
+#. module: sale_pricelist_technical
+#: model:ir.model,name:sale_pricelist_technical.model_sale_order
+msgid "Sale Order"
+msgstr "Bon de commande"
+
+#. module: sale_pricelist_technical
+#: model:product.pricelist,name:sale_pricelist_technical.technical_pricelist
+msgid "Technical Pricelist"
+msgstr "Liste de prix technique"
+
+#. module: sale_pricelist_technical
+#: model:ir.model.fields,help:sale_pricelist_technical.field_res_partner__property_product_pricelist
+#: model:ir.model.fields,help:sale_pricelist_technical.field_res_users__property_product_pricelist
+msgid "This pricelist will be used, instead of the default one, for sales to the current partner"
+msgstr "Cette liste de prix sera utilisée, à la place de la liste par défaut, pour les ventes faites au partenaire actuel."
+
```

Comment:
I couldn't find the source text in the code.

Reply:
Hi. It's a text that is autogenerated by Odoo when generating new translation.
I don't know why exactly. The module overload the field `property_product_pricelist` but doesn't define help value...
I removed it, but maybe weblate will add it again !